frappe/desk/doctype/document_template/document_template.js

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2022, Frappe Technologies and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Document Template', {
+	// refresh: function(frm) {
+
+	// }
+});
```

Comment:
Let's prevent unnecessary execution. Perhaps we should make a PR to default to this.

```suggestion
// frappe.ui.form.on('Document Template', {
//	 refresh: function(frm) {

//	 }
// });
```